index.d.ts

```diff
@@ -788,10 +783,191 @@ type ExecaCommonError = {
+
+	/**
+	Whether the subprocess was terminated by a signal (like `SIGTERM`) sent by either:
+	- The current process.
+	- Another process. This case is [not supported on Windows](https://nodejs.org/api/process.html#signal-events).
+	*/
+	isTerminated: CommonResult<IsSync, OptionsType>['isTerminated'];
+
+	/**
+	The name of the signal (like `SIGTERM`) that terminated the subprocess, sent by either:
+	- The current process.
+	- Another process. This case is [not supported on Windows](https://nodejs.org/api/process.html#signal-events).
+
+	If a signal terminated the subprocess, this property is defined and included in the error message. Otherwise it is `undefined`.
+	*/
+	signal?: CommonResult<IsSync, OptionsType>['signal'];
+
+	/**
+	A human-friendly description of the signal that was used to terminate the subprocess. For example, `Floating point arithmetic error`.
+
+	If a signal terminated the subprocess, this property is defined and included in the error message. Otherwise it is `undefined`. It is also `undefined` when the signal is very uncommon which should seldomly happen.
+	*/
+	signalDescription?: CommonResult<IsSync, OptionsType>['signalDescription'];
+
+	/**
+	The current directory in which the command was run.
+	*/
+	cwd: CommonResult<IsSync, OptionsType>['cwd'];
+
+	/**
+	Duration of the subprocess, in milliseconds.
+	*/
+	durationMs: CommonResult<IsSync, OptionsType>['durationMs'];
+
+	/**
+	Whether the subprocess was canceled using the `cancelSignal` option.
+	*/
+	isCanceled: CommonResult<IsSync, OptionsType>['isCanceled'];
+
+	/**
+	The output of the subprocess with `stdout` and `stderr` interleaved.
+
+	This is `undefined` if either:
+	- the `all` option is `false` (default value)
+	- both `stdout` and `stderr` options are set to [`'inherit'`, `'ignore'`, `Stream` or `integer`](https://nodejs.org/api/child_process.html#child_process_options_stdio)
+
+	This is an array if the `lines` option is `true`, or if either the `stdout` or `stderr` option is a transform in object mode.
+	*/
+	all: CommonResult<IsSync, OptionsType>['all'];
+
+	/**
+	Results of the other subprocesses that were piped into this subprocess. This is useful to inspect a series of subprocesses piped with each other.
+
+	This array is initially empty and is populated each time the `.pipe()` method resolves.
+	*/
+	pipedFrom: CommonResult<IsSync, OptionsType>['pipedFrom'];
+}
+
+/**
+Exception thrown when the subprocess [fails](#failed), either:
+- its exit code is not `0`
+- it was terminated with a signal, including `.kill()`
+- timing out
+- being canceled
+- there's not enough memory or there are already too many subprocesses
+
+This has the same shape as successful results, with a few additional properties.
+*/
+export class ExecaError<OptionsType extends Options = Options> extends CommonErrorClass<false, OptionsType> {
+	readonly name: 'ExecaError';
+}
+
+/**
+Exception thrown when the subprocess [fails](#failed), either:
```

Comment:
The link needs to be absolute.

Reply:
Fixed, keeping it consistent with the rest of the file, which only includes external links, as opposed to links inside the `readme.md`.